plugin-api/src/main/java/org/hyperledger/besu/plugin/services/metrics/PoaQueryService.java

```diff
@@ -22,12 +22,12 @@
 /**
  * Provides relevant data for producing metrics on the status of a Proof of Authority (PoA) node.
  */
-public interface PoAMetricsService {
+public interface PoaQueryService {
```

Comment:
This is a backwards incompatible change on our public, stable plugin APIs.  I think we need to deprecate the old interface but keep it around (the same implementation would implement both old and new).  We also need to document a deprecation policy so we are clear about when we can remove the old interface.

Reply:
done